portfolio.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+    <link type="text/css" rel="stylesheet" href="normalize.css">
+    <link type="text/css" rel="stylesheet" href="styles.css">
+</head>
+
+<!--[if lte IE 9]><body class="ieLegacy"><![endif]-->
+<!--[if !IE]><body><![endif]-->
+
+<body>
+    <div>
+        <header class="container">
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                </ul>
+            </nav>
+
+            <h1>My Portfolio</h1>
+        </header>
+
+        <main>
+            <div class="col3 col2">
+                <h2><a href="/sites/academy-php7/html/Responsive site project/index.html">Pilot Shop</a></h2>
```

Comment:
need to change all links from absolute to relative within your project folders as these links wont work on the server

Reply:
As far as I can see these links are positioned relatively from ".col3 a", the only link positioned absolutely is the contact icon, where the link is positioned according to the icon, will these need to be changed? this was the only thing that had to be fixed in IE9, so might need to check it with browser stack again if this needs changing.